Readme.md

```diff
@@ -59,6 +60,21 @@ For information about terms used in this document see: [terminology](./docs/term
 npm install commander
 ```
 
+## Quick Start
+
+```js
+const { program } = require("commander");
+
+program
+    .command('list') // The name of your command
```

Comment:
I see potential for confusion in the comment for new user between giving the program a name, and adding a new subcommand?

Perhaps: add a subcommand named "list"

Reply:
Yeah I see